source/help/apps/desktop-guide.rst

```diff
@@ -0,0 +1,176 @@
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Flash taskbar icon when a new message is received                             | v1.3.0 +                  | N/A                       | v1.4.0 +                  | `MetalCar <https://github.com/metalcar>`_ (Windows); `jnugh <https://github.com/jnugh>`_ (Linux)                                                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+
+**Hide menu bar**
+
+Hides the menu bar. When enabled, pressing ``ALT`` will toggle whether the menu bar is shown or hidden.
+
+This setting is off by default.
+
+**Allow mixed content**
+
+If your server is hosted on `https://`, insecure content, images and scripts with `http://` are not rendered by default. This option allows such content to be rendered. If you do enable it, please be mindful of potential security risks shared over `http://` protocols.
+
+This setting is on by default.
+
+Note: Enabling both secure and insecure content will break YouTube videos. An open issue for ``electron`` on which the app is built is under investigation: https://github.com/electron/electron/issues/2749
+
+**Show icon on Menu Bar** (OS X)
+
+When enabled, a black dot indicating unread messages is displayed on the team tab bar.
+
+This setting is off by default.
+
+**Start app on login**
+
+When enabled, Mattermost application starts when you login to your machine.
+
+This setting is on by default.
+
+**Toggle window visibility when clicking on the tray icon** (Windows)
+
+When enabled, clicking on the system tray icon allows you to toggle the window open and minimized.
+
+This setting is off by default.
+
+**Leave app running in notification center when application window is closed.** (OS X, Linux)
+
+When enabled, closing the application window will leave the Mattermost desktop app running in your notification center. This can be useful if you’d like to check for unread mentions while away from the app.
+
+This setting is off by default.
+
+**Show red badge on taskbar icon to indicate unread messages** (Windows, OS X)
+
+If you have multiple active teams (either on a single or multiple servers), having a red badge for unread messages may be annoying as it would likely be displayed most of the time. You may use this setting to configure when the red badge is shown on the taskbar icon. 
```

Comment:
Propose removing the pre-amble, and leaving just the explanation below. The pre-amble uses the word "annoying" and seems to cast the product in a negative light.